tools/tune/eq/eq_deploy_to_dut.m

```diff
@@ -0,0 +1,27 @@
+function status = eq_deploy_to_dut(target, temp_file)
+% deploy eq over ssh to a given dut with sof-eqctl
+% the end target is expected to be some sort of unix system with sof-eqctl
+% already installed. The system should also have ssh keys installed to prevent
+% the toolchain from logging in
+
+ip = target.ip;
+user = target.user;
+dev = target.device;
+ctrl = target.control;
+
+if isunix() || ismac()
+	transfer_template = "scp %s %s@%s:/tmp/sof-eq-config.txt";
+	config_template = "ssh -t %s@%s sof-eqctl -Dhw:%d -n %d -s /tmp/sof-eq-config.txt";
+	transfer_command = sprintf(transfer_template, temp_file, user, ip);
+	config_command = sprintf(config_template, ip, user, dev, ctrl);
```

Comment:
Here's a mistake that breaks the latter command with "ip@user" attempt.

Reply:
Fixed, thanks for spotting that, GLK is broken for me right now so I can't test that part.